testing/SendLog/SendLog.py

```diff
@@ -0,0 +1,95 @@
+## Hrishi Hiraskar
+## 8 September 2016
+
+import gevent
+import time
+from gevent import monkey
+from gevent.pywsgi import WSGIServer
+from flask import Flask, request, Response, render_template
+
+monkey.patch_all()
+
+app = Flask(__name__)
+
+# Delay time to look for new line (in s)
+LOOK_DELAY = 0.01	
+
+def get_line(file):
+	# Function to get a new line from file
+	line = file.readline()
+	while not line:
+		# If no new line is found
+		# Wait for some time and look again
+		gevent.sleep(LOOK_DELAY)
+		line = file.readline()
+	return line
+		
+def parse_line(line):
+	# Function to parse the line
+	# Returns tuple of figure ID and state
+	# state = 0 if new figure is created
+	#         1 if current fig end
+	#         2 otherwise
+	linewords = line.split(' ')
+	if linewords[0] == "Initialization":
+		# New figure created
+		# Get fig id
+		figureID = int(linewords[-1])
```

Comment:
Almost skipped this one!

Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.

Similarly for `logfile` and `figureList`